recipe/meta.yaml

```
@@ -10,29 +10,29 @@ source:
 
 build:
   skip: True  # [py<37]
-  number: 0
+  number: 1
   script:
-    - python -c '__import__("os").unlink("pyproject.toml")'
-    - {{ PYTHON }} -m pip install . -vv
+    - {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
 
 requirements:
   host:
     - python
     - pip
     - wheel
-    - poetry
+    - poetry-core >=1.0.0
```

```suggestion
    - poetry_core >=1,<2
```
see https://github.com/graphql-python/graphql-core/blob/v3.2.3/pyproject.toml#L88C24-L88C30